packages/main/src/components/FilterBar/FilterBar.jss.ts

```diff
@@ -28,35 +28,16 @@ const styles = ({ theme, contentDensity, parameters }: JSSTheme) => {
       flexWrap: 'wrap',
       paddingTop: '1rem',
       paddingBottom: '1rem',
-      background: parameters.sapUiObjectHeaderBackground
-    },
-    '@media (max-width: 599px)': {
-      filterArea: {
-        columnCount: 3,
-        columnGap: '0.5rem',
-        rowGap: '0.5rem'
-      }
-    },
-    '@media (min-width: 600px) and (max-width: 1023px)': {
-      filterArea: {
-        columnCount: 8,
-        columnGap: '1rem',
-        rowGap: '1rem'
-      }
+      background: parameters.sapUiObjectHeaderBackground,
+      transition: 'max-height 0.2s ease-out, opacity 0.2s ease-in'
     },
-    '@media (min-width: 1024px) and (max-width: 1439px)': {
-      filterArea: {
-        columnCount: 12,
-        columnGap: '1rem',
-        rowGap: '1rem'
-      }
+    filterAreaClosed: {
+      maxHeight: '0',
+      opacity: 0
```

Comment:
I would add `padding: 0` as well, otherwise there is lots of whitespace in the bottom.